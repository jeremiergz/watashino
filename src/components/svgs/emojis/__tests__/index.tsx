import NerdIcon from '@/components/svgs/emojis/Nerd';
import SadIcon from '@/components/svgs/emojis/Sad';
import ReactTestRenderer from 'react-test-renderer';

describe('@/components/svgs/emojis Test Suite', () => {
  describe('Happy Test Suite', () => {
    it('matches snapshot', () => {
      const jsx = <NerdIcon />;
      expect(ReactTestRenderer.create(jsx).toJSON()).toMatchSnapshot();
    });
  });

  describe('Sad Test Suite', () => {
    it('matches snapshot', () => {
      const jsx = <SadIcon />;
      expect(ReactTestRenderer.create(jsx).toJSON()).toMatchSnapshot();
    });
  });
});
