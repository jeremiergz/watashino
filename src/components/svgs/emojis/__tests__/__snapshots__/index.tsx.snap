// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@/components/svgs/emojis Test Suite Happy Test Suite matches snapshot 1`] = `
<svg
  className=""
  height={32}
  viewBox="0 0 72 72"
  width={32}
  xmlns="http://www.w3.org/2000/svg"
>
  <g
    id="color"
  >
    <circle
      cx="36"
      cy="36"
      fill="#fcea2b"
      r="23"
    />
    <path
      d="M32.1434,47.3742c-0.282,1.7234-0.4344,5.4883,3.8566,4.64v-4.3697"
      fill="#ffffff"
    />
    <path
      d="M39.8566,47.3742c0.282,1.7234,0.4344,5.4883-3.8566,4.64v-4.3697"
      fill="#ffffff"
    />
  </g>
  <g
    id="hair"
  />
  <g
    id="skin"
  />
  <g
    id="skin-shadow"
  />
  <g
    id="line"
  >
    <circle
      cx="36"
      cy="36"
      fill="none"
      r="23"
      stroke="#000000"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeMiterlimit="10"
      strokeWidth="2"
    />
    <path
      d="M19.074,30.4073c0,0-0.953,8.5508,6.8608,7.9185c2.6197-0.212,7.8164-0.6507,7.867-8.3427 c0.0046-0.6979-0.0078-1.5989-0.8108-2.6298c-1.0647-1.3669-3.5716-1.9711-9.9446-1.422 C23.0465,25.9314,19.6002,25.8323,19.074,30.4073z"
      fill="none"
      stroke="#000000"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeMiterlimit="10"
      strokeWidth="2"
    />
    <polyline
      fill="none"
      points="19.1447,29.8742 18.7117,26.5029 22.5446,25.9752"
      stroke="#000000"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeMiterlimit="10"
      strokeWidth="2"
    />
    <path
      d="M52.9325,30.4073c0,0,0.953,8.5508-6.8608,7.9185c-2.6197-0.212-7.8164-0.6507-7.867-8.3427 c-0.0046-0.6979,0.0078-1.5989,0.8108-2.6298c1.0647-1.3669,3.5716-1.9711,9.9446-1.422 C48.9601,25.9314,52.4064,25.8323,52.9325,30.4073z"
      fill="none"
      stroke="#000000"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeMiterlimit="10"
      strokeWidth="2"
    />
    <path
      d="M31.6962,26.3593c0,0,4.1241,2.5339,8.6569,0"
      fill="none"
      stroke="#000000"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeMiterlimit="10"
      strokeWidth="2"
    />
    <path
      d="M33.8022,31.2614c0,0,2.1275-3.7509,4.383,0"
      fill="none"
      stroke="#000000"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeMiterlimit="10"
      strokeWidth="2"
    />
    <polyline
      fill="none"
      points="52.8552,29.8767 53.2883,26.5054 49.4554,25.9777"
      stroke="#000000"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeMiterlimit="10"
      strokeWidth="2"
    />
    <path
      d="M34.1463,29.9706c0,0,1.7954-3.7509,3.6988,0"
      fill="none"
      stroke="#000000"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeMiterlimit="10"
      strokeWidth="2"
    />
    <path
      d="M32.1434,47.3742 c-0.282,1.7234-0.4344,5.4883,3.8566,4.64v-4.3697"
      fill="none"
      stroke="#000000"
      strokeMiterlimit="10"
      strokeWidth="2"
    />
    <path
      d="M30,31c0,1.6568-1.3448,3-3,3c-1.6553,0-3-1.3433-3-3c0-1.6552,1.3447-3,3-3C28.6552,28,30,29.3448,30,31"
    />
    <path
      d="M48,31c0,1.6568-1.3447,3-3,3s-3-1.3433-3-3c0-1.6552,1.3447-3,3-3S48,29.3448,48,31"
    />
    <path
      d="M39.8566,47.3742 c0.282,1.7234,0.4344,5.4883-3.8566,4.64v-4.3697"
      fill="none"
      stroke="#000000"
      strokeMiterlimit="10"
      strokeWidth="2"
    />
    <path
      d="M45.8148,44.8055c-6.574,3.5248-14.0454,3.6576-19.6295,0"
      fill="none"
      stroke="#000000"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeMiterlimit="10"
      strokeWidth="2"
    />
  </g>
</svg>
`;

exports[`@/components/svgs/emojis Test Suite Sad Test Suite matches snapshot 1`] = `
<svg
  className=""
  height={32}
  viewBox="0 0 72 72"
  width={32}
  xmlns="http://www.w3.org/2000/svg"
>
  <g
    id="line-supplement"
  >
    <path
      d="m52 52.38c0.8775-1.631 1-5.38 1-7.38 0-4-4-11-4-11"
      fill="none"
      stroke="#000"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeMiterlimit="10"
      strokeWidth="2"
    />
    <path
      d="m44 36s3 6.728 3 10c0 3.065-1 8-1 10"
      fill="none"
      stroke="#000"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeMiterlimit="10"
      strokeWidth="2"
    />
    <path
      d="m20 52.4c-0.8775-1.631-1-5.4-1-7.4 0-4 4-11 4-11"
      fill="none"
      stroke="#000"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeMiterlimit="10"
      strokeWidth="2"
    />
    <path
      d="m28 36s-3 6.728-3 10c0 3.065 1 8 1 10"
      fill="none"
      stroke="#000"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeMiterlimit="10"
      strokeWidth="2"
    />
  </g>
  <g
    id="color"
  >
    <path
      d="m36.2 13.32c-12.57 0-22.8 10.23-22.8 22.8 0 12.57 10.23 22.8 22.8 22.8 12.57 0 22.8-10.23 22.8-22.8 0-12.57-10.23-22.8-22.8-22.8z"
      fill="#fcea2b"
    />
    <path
      d="m35.93 51.58c-2.461 0-4.742-2.368-4.742-4.987 0-2.619 2.444-4.66 4.905-4.66s4.701 2.205 4.701 4.823c-1e-6 2.619-2.403 4.823-4.864 4.823z"
      fillRule="evenodd"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeMiterlimit="10"
      strokeWidth="0.1"
    />
    <path
      d="m31.37 45.29c2.025 1.288 7.318 1.288 9.26 0l-4.63-4.005z"
      fill="#fff"
      strokeLinejoin="round"
      strokeWidth="2"
    />
    <path
      d="m29.7 32.02c-5.762 9.541-3.86 14.27-3.696 23.98 0 1.803-5.146-2.412-6-4-2.17-5.92-0.3989-13.81 2.5-19.18 2.887-1.622 6.992-2.084 7.196-0.8031z"
      fill="#92d3f5"
    />
    <path
      d="m42.3 32.4c5.762 9.541 3.86 14.27 3.696 23.98 0 1.803 5.146-2.412 6-4 2.17-5.92 0.3989-13.81-2.5-19.18-2.887-1.622-6.992-2.084-7.196-0.8031z"
      fill="#92d3f5"
    />
  </g>
  <g
    id="hair"
  />
  <g
    id="skin"
  />
  <g
    id="skin-shadow"
  />
  <g
    id="line"
  >
    <path
      d="m40.5 32.82c0.4132-0.344 2.246-1.792 4.909-1.636 2.161 0.1259 3.61 1.233 4.091 1.636"
      fill="none"
      stroke="#000"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeMiterlimit="10"
      strokeWidth="2"
    />
    <path
      d="m22.5 32.82c0.4132-0.344 2.246-1.792 4.909-1.636 2.161 0.1259 3.61 1.233 4.091 1.636"
      fill="none"
      stroke="#000"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeMiterlimit="10"
      strokeWidth="2"
    />
    <circle
      cx="36"
      cy="36"
      fill="none"
      r="23"
      stroke="#000"
      strokeMiterlimit="10"
      strokeWidth="2"
    />
    <circle
      cx="36"
      cy="46.6"
      fill="none"
      fillRule="evenodd"
      r="4.759"
      stroke="#000"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeMiterlimit="10"
      strokeWidth="2"
    />
  </g>
</svg>
`;
